README.md

```diff
@@ -189,6 +189,24 @@ struct will have the same field tagged `#[serde(skip_serializing_if = "Option::i
 This attribute makes serde skip fields entirely if the value of the `Option` is
 none (rather than saving e.g. `"value" = null` if serializing to json).
 
+7. Skip a field entirely:
+
+```rust
+#[optional_struct]
+#[derive(Default)]
+struct Foo {
+    #[optional_skip]
+    bar: char,
+    baz: bool,
+}
+
+fn main() {
+    let opt_f = OptionalFoo { baz: Some(false) };
+}
+```
+
+If the `optional_skip` attribute is used, `Default` is used to fill in missing fields in the ``TryFrom` implementation.   Therefore Default must be implemented on the struct.
```

Comment:
While more work to implement, would it make sense to instead have `optional_default`, which generates a `bar = Default::default()` this removes the `Default` constraint on `Foo` itself.

Or do I misunderstand your usecase?

Reply:
The use case is that the Foo struct has an attribute, but OptionalFoo struct does not have the attribute.

The reason for this is that OptionalFoo is a struct exposed via API and config file, and Foo is internal, so "optional_skip" is used for internal-only attributes.